nixos/doc/manual/release-notes/rl-2111.section.md

```diff
@@ -11,5 +11,52 @@ In addition to numerous new and upgraded packages, this release has the followin
 ## Backward Incompatibilities
 
 * The `staticjinja` package has been upgraded from 1.0.4 to 2.0.0
+* The `paperless` module and package have been removed. All users should migrate the to
+  successor `paperless-ng` instead. The Paperless project [has been
+  archived](https://github.com/the-paperless-project/paperless/commit/9b0063c9731f7c5f65b1852cb8caff97f5e40ba4)
+  and advices all users to use `Paperless-ng` instead.
```

Comment:
Thanks @andir!

(Looks good, just one nitpick: I think this should be ‘advi**s**es’)

Reply:
Oh, good catch. I'll update